GetJobber/help_theme_2019
# Add webinar bar

This adds the webinar bar to the help site, and a script to show it only on certain pages. 

To test:

- Build the helpsite templates and styles from the branch and preview
- Check to see that the webinar bar is displaying on the pages in the [Asana task description](https://app.asana.com/0/1164538569125870/1200019927336031) (a spot check should be ok)
- Check to see that the webinar bar looks like the screenshots in desktop and mobile

![Screen Shot 2021-03-09 at 2 10 23 PM](https://user-images.githubusercontent.com/9054824/110540757-1d746c00-80e4-11eb-862f-c3a58bd8229e.png)
![Screen Shot 2021-03-09 at 2 10 07 PM](https://user-images.githubusercontent.com/9054824/110540758-1e0d0280-80e4-11eb-95bf-328a4a3a76ed.png)
